electron/anticheat/titles_map.ts

```diff
@@ -0,0 +1,27 @@
+// this hash contains a mapping of titles as displayed in Epic to titles as
+// displayed in areweanticheatyet.com https://github.com/Starz0r/AreWeAntiCheatYet/blob/master/games.json
+//
+// comparison is case insensitive, so no need to add cases like 'VALORANT': 'Valorant'
+// if the epic title is contained, so no need to add cases like 'Fall Guts': 'Fall Guys: Ultimate Knockout'
+//
+// left is the epic title, right is the areweanticheatyet title
+const TITLES_MAP: { [key: string]: string } = {
+  'Battlefield™ 2042': 'Battlefield 2042',
```

Comment:
You could also consider stripping out any ™ characters from the comparison

Reply:
I thought about that, but I didn't want to start stripping things out because I don't know if that can generate any issue in the future.

So I went with a more strict approach for now, since we still need to cover cases where stripping characters is not enough like `'STAR WARS™ Battlefront™ Ultimate Edition'` and `'Star Wars: Battlefront (2015)'`. If, in the future, we see it's too hard to maintain this list maybe we can find the right things to remove.